internal/manifests/manifests.go

```diff
@@ -54,7 +54,7 @@ func BuildAll(params manifestutils.Params) ([]client.Object, error) {
 	var manifests []client.Object
 	manifests = append(manifests, configMaps)
 	if params.Tempo.Spec.ServiceAccount == naming.DefaultServiceAccountName(params.Tempo.Name) {
-		manifests = append(manifests, serviceaccount.BuildDefaultServiceAccount(params.Tempo))
+		manifests = append(manifests, serviceaccount.BuildDefaultServiceAccount(params.Tempo, params.StorageParams))
```

Comment:
How about passing just `params`? Internally the function can access the `Tempo` and `StorageParams` instances.

Reply:
it makes sense, other  methods uses params